personalization-webcomponents/src/checklist-domi.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div>
+    <!-- eslint-disable-next-line vue/no-v-html -->
+    <div v-html="mucIconsSprite" />
+    <!-- eslint-disable-next-line vue/no-v-html -->
+    <div v-html="customIconsSprite" />
+    <div v-html="customIconsSprite" />
+
+      <div v-if="loading">
+        <skeleton-loader
+            v-for="elem in [1, 2, 3, 4]"
+            :key="elem"
+        >
+        </skeleton-loader>
+      </div>
+      <div v-else>
+        <checklist-header
+            v-for="(checklist, index) in checklists"
```

Comment:
you should only show one header - as the checklist-detail should show the details of only one checklist with id you get from the query param https://github.com/it-at-m/dbs/blob/main/personalization-webcomponents/src/util/constants.ts#L3